libs/application/templates/parental-leave/src/fields/PeriodPercentage/PeriodPercentage.tsx

```diff
@@ -58,13 +62,9 @@ export const PeriodPercentage: FC<PeriodPercentageField> = ({
 
   const fieldId = `periods[${currentIndex}].ratio`
 
-  let error
-
-  if (errors?.periods?.[currentIndex]?.ratio?.message) {
-    error = errors?.periods?.[currentIndex]?.ratio?.message
-  } else if (errors?.[fieldId]) {
-    error = errors?.[fieldId]
-  }
+  const error =
+    getErrorViaPath(errors, `periods.[${currentIndex}].ratio?.message`) ??
+    getErrorViaPath(errors, 'fieldId')
```

Comment:
This should be variable 'fieldId', right?

Reply:
Correct, well spotted!